src/wallets/IdentityWallet.ts

```diff
@@ -480,3 +484,22 @@ export function calculateIdentityAddress(
       .slice(2 + 2 * 12)
   return toChecksumAddress(address)
 }
+
+/**
+ * Generates a random nonce to use for meta transactions.
+ * The nonce fits into the range of ]2^255, 2^256[.
+ * This is an alternative to the up counting nonce (]0, 2^255[) without the need
+ * to query a [[TLProvider]].
+ */
+export function getRandomNonce(): string {
+  const random = BigNumber.random()
+  const nonce = random.multipliedBy(RANDOM_NONCE_RANGE).plus(MIN_RANDOM_NONCE)
```

Comment:
The range is size `2**255`, you should use enough randomness so that the full capacity of the range is used: `random = BigNumber.random(77)`. I chose 77 because `2*255 < 10**17`

see: https://mikemcl.github.io/bignumber.js/#random